### PR TITLE
[DOC]Add Pachyderm Hub docs for GA

### PR DESCRIPTION
**Pull request by svekars**

### CODE REVIEW

File: doc/docs/master/hub/manage-entities/manage-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Each workspace has isolated computing, processing, storage, and other
+resources. Resource autoscaling is managed by Pachyderm behind the
+scenes. You cannot move data or pipelines between workspaces.
+Each workspace in an organization must have a unique name.
+
+## Create a Workspace
+
+You need a workspace to star building your pipelines and working
+with Pachyderm.
```
Comment (JoeyZwicker) [inline doc/docs/master/hub/manage-entities/manage-workspaces.md:18]:
```suggestion
```suggestion
You need a workspace to start adding data and building pipelines within Pachyderm
```
```

File: doc/docs/master/hub/manage-entities/manage-workspaces.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Each workspace in an organization must have a unique name.
+
+## Create a Workspace
+
+You need a workspace to star building your pipelines and working
+with Pachyderm. 
+
+To create a workspace, complete the following steps:
+
+1. Click **Create New Workspace** or **Create a 4-Hour Workspace**.
```
Comment (JoeyZwicker) [inline doc/docs/master/hub/manage-entities/manage-workspaces.md:22]: This is cleaner and simpler than the "create a workspace" flow you have in the getting started section. I think we should update the other one to be simpler like this

File: doc/docs/master/hub/hub_getting_started.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+| Memory     | 12 GB      |
+| GPU        | None       |
+| Time limit | 4 hours    |
+
+
+To create a workspace, complete the following steps:
+
+1. If you have not yet done so, log in to Pachyderm Hub.
+1. Click **Create New Workspace**.
+1. Type a name for your cluster. For example, `test1`.
```
[the remark this answers is not in the document]
Comment (JoeyZwicker) [reply doc/docs/master/hub/hub_getting_started.md:67]: See note in "workspace" specific doc. That seems like a simpler number of steps.

File: doc/docs/master/hub/manage-entities/manage-workspaces.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+1. Fill out the required fields and accept Terms and Conditions.
+1. Click **Create**.
+
+   Your workspace should be created instantly.
```
Comment (JoeyZwicker) [inline doc/docs/master/hub/manage-entities/manage-workspaces.md:26]: inclue note from getting started about "pre-warmed" clusters

File: doc/docs/master/hub/manage-entities/manage-workspaces.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Connect to Your Workspace
+
+You can access your workspace either by using the command-line interface (CLI)
+called `pachctl` or by using the PachDash UI. Although some of the
```
Comment (JoeyZwicker) [inline doc/docs/master/hub/manage-entities/manage-workspaces.md:31]: Should not mention UI is most of this flow. They MUST connect via pachctl to get any real use out of the system. Dash is a small bonus that we can mention later. 

Use the good phrasing I mentioned in getting started here. No need to write basically the same info twice differently.

File: doc/docs/master/hub/manage-entities/manage-workspaces.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
+# Manage Workspaces
```
Comment (JoeyZwicker) [inline doc/docs/master/hub/manage-entities/manage-workspaces.md:1]: This doc does not have any section describing resources and autoscaling in Hub Pro. While there are not a lot of options for users to adjust here, it should describe how those are handled and managed and the limitations.

File: doc/docs/master/hub/manage-entities/manage-members.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+# Manage Members
```
Comment (JoeyZwicker) [inline doc/docs/master/hub/manage-entities/manage-members.md:1]: This whole doc can be moved to be part of the orgs doc as members only are relevant in context of orgs.

File: doc/docs/master/hub/manage-entities/manage-members.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+A member is a user account in Pachyderm Hub that represents
+a person interacting with Pachyderm Hub. A member is associated with a
+GitHub or Google account. Therefore, you must have either of these
+to log in to Pachyderm Hub. All users in Pachyderm Hub get administrative
+access to the corresponding Pachyderm Hub organization and all workspaces.
+In the current Pachyderm version, other roles are not supported.
+The first user that has created the organization is responsible for
+billing. This role can later be transferred to other users.
```
Comment (JoeyZwicker) [inline doc/docs/master/hub/manage-entities/manage-members.md:10]:
```suggestion
With an Organization, each uniquely authenticated GitHub or Google account is considered a Member.  All Members in Pachyderm Hub get administrative
access to the corresponding Pachyderm Hub organization and all workspaces.
In the current Pachyderm Hub version, other roles are not yet supported.
The first user that has created the organization is responsible for
billing. This role can later be transferred to other users.
```

**Comment by nadegepepin** [comment]
Closing this after meeting with Jonathan. 
The **Getting Started** section of the doc already has a more updated version of **Getting Started with Hub** that details the steps to spin a cluster, install pachctl, and set a context. The 3 additional pages (manage workspace, org, members) are not of use. The upgrade from a free account to a paying account is as simple as clicking the "Upgrade" button and picking a plan.  The creation/deletion of workspaces is self-explanatory, as is the creation/deletion of team members. In any case, if there was a need for a help page, that page should be part of Hub itself.

**Comment by nadegepepin** [comment]
This PR was closed as it was not relevant anymore. However, it led to an update of the Getting started with Hub page See PR #5775

**Pull request closed without merging**